packages/migrate/src/__tests__/MigrateReset.test.ts

```diff
@@ -1,6 +1,9 @@
 process.env.PRISMA_MIGRATE_SKIP_GENERATE = '1'
 process.env.GITHUB_ACTIONS = '1'
 
+// TODO: Windows: some snapshot tests fail on Windows because of emoji.
+const testIf = (condition: boolean) => (condition ? test : test.skip)
```

Comment:
Can't we filter these out from the snapshots? Might be as dumb as really filtering out the x ones we use hard coded.

Reply:
Yes, we can remove emoji from all snapshots and filter them out in the snapshot serializer to make the tests ignore them on Linux and macOS. Not the other way unfortunately. Should we do that?

Reply:
I think that should be fine.

Reply:
Resolving this comment as that would probably be out of scope of this PR for now. I'll create an issue for the `TODO: Windows` comments. This will be one of the lower-hanging fruits to fix among those.